src/doc/unstable-book/src/language-features/loop-break-value.md

```diff
@@ -4,7 +4,89 @@ The tracking issue for this feature is: [#37339]
 
 [#37339]: https://github.com/rust-lang/rust/issues/37339
 
+Documentation to be appended to section 3.6 of the book: Loops (after "Loop Labels", or before if
+the "Break" section is moved). If this is deemed too complex a feature this early in the book, it
+could also be moved to a new section (please advise). This would allow examples breaking with
+non-primitive types, references, and discussion of coercion (probably unnecessary however).
```

Comment:
So, our current strategy is that we have an appendix G, where stuff can land at first https://doc.rust-lang.org/nightly/book/second-edition/appendix-07-newest-features.html

I'd recommend saying this should go there instead